.travis.yml

```diff
@@ -83,7 +85,10 @@ install:
   - npm install
 
 script:
-  - if [[ "$PLATFORM" =~ local ]]; then npm run test-browser; fi
-  - if [[ "$PLATFORM" =~ local ]]; then npm run test-ios; fi
-  - if [[ "$PLATFORM" =~ TODO ]]; then npm run test-android; fi
-  - if [[ "$PLATFORM" != local ]]; then node main.js --config ./pr/$PLATFORM --plugin ./spec/testable-plugin/ --shouldUseSauce --buildName travis-paramedic-$TRAVIS_JOB_NUMBER; fi
+  - if [[ "$PLATFORM" =~ local ]]; then
+      node main.js --config ./pr/$PLATFORM --plugin ./spec/testable-plugin/ --buildName travis-paramedic-$TRAVIS_JOB_NUMBER;
+    fi
+
+  - if [[ "$PLATFORM" != local* ]]; then 
+      node main.js --config ./pr/$PLATFORM --plugin ./spec/testable-plugin/ --buildName travis-paramedic-$TRAVIS_JOB_NUMBER --shouldUseSauce;
```

Comment:
Is `--shouldUseSauce` really required or could this be part of the config file? That would enable us to get rid of those `if`s.

Reply:
Indeed, `shouldUseSauce` can be defined in configuration as everything else.